### PR TITLE
Data release: 2021_09_21_01_01_53

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: test/integration/golden_response/get_statvar_summary/total_crimes.json [hunk 1]
```diff
@@ -82,6 +82,126 @@
             }
           ]
         }
+      },
+      "provenanceSummary": {
```
Comment (shifucun) [inline test/integration/golden_response/get_statvar_summary/total_crimes.json:86]: @n-h-diaz Please confirm the cache change

File: test/integration/golden_response/get_statvar_group_node/root_mtv_jpn.json [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
       "displayName": "Demographics",
-      "numDescendentStatVars": 4102
+      "numDescendentStatVars": 4144
     },
     {
       "id": "dc/g/Household",
       "specializedEntity": "Household",
       "displayName": "Household",
-      "numDescendentStatVars": 1000
+      "numDescendentStatVars": 1223
```
Comment (shifucun) [inline test/integration/golden_response/get_statvar_group_node/root_mtv_jpn.json:14]: @pradh Please confirm the increased children

File: test/integration/golden_response/get_place_page_data/city.sample.json [hunk 3]
```diff
@@ -7663,7 +7678,8 @@
             "2014": 16.6,
             "2015": 16.7,
             "2016": 15.2,
-            "2017": 15.7
+            "2017": 15.7,
+            "2018": 15.8
```
Comment (pradh) [inline test/integration/golden_response/get_place_page_data/city.sample.json:7682]: This is CDC Places data lining up with prior CDC 500

File: test/integration/golden_response/get_statvar_group_node/root_mtv_jpn.json (hunk 2 above)
Comment (pradh) [reply test/integration/golden_response/get_statvar_group_node/root_mtv_jpn.json:14]: Yes, that's from cl/394700399

**Review by pradh — APPROVED** [review]
Looks good to me

File: test/integration/golden_response/get_statvar_group_node/root_mtv.json [hunk 4]
```diff
@@ -74,7 +74,7 @@
       "id": "dc/g/Uncategorized",
       "specializedEntity": "Uncategorized",
       "displayName": "Uncategorized",
-      "numDescendentStatVars": 10
+      "numDescendentStatVars": 168
```
Comment (beets) [inline test/integration/golden_response/get_statvar_group_node/root_mtv.json:77]: there's quite a few more here to curate
fyi/ @ajaits

**Review by beets — APPROVED** [review]

File: test/integration/golden_response/get_statvar_summary/total_crimes.json (hunk 1 above)
Comment (n-h-diaz) [reply test/integration/golden_response/get_statvar_summary/total_crimes.json:86]: this looks okay, thanks!

**Review by n-h-diaz — APPROVED** [review]

**Comment by shifucun** [comment]
Thank you all. Will submit and start the release.

**Pull request merged**